app/services/s3_query_service.rb

```diff
@@ -119,9 +119,6 @@ def find_s3_file(filename:)
   # Retrieve the S3 resources uploaded to the S3 Bucket
   # @return [Array<S3File>]
   def client_s3_files(reload: false, bucket_name: self.bucket_name)
-    # Allow migration objects to load locally with an AWS key
-    return [] if Rails.env.development?
-
```

Comment:
We probably missed this during the refactoring...but that throw me off for a while! 😱